<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hotkeysWindow</class>
 <widget class="QDialog" name="hotkeysWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="upLabel">
     <property name="text">
      <string>Up:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="upHotkeyButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QPushButton" name="speedDecreaseHotkeyButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="pitchDownLabel">
     <property name="text">
      <string>Pitch Down:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="pitchDownHotkeyButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="straftLeftHotkeyButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QLabel" name="speedDecreaseLabel">
     <property name="text">
      <string>Speed Decrease:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="downHotkeyButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="straftLeftLabel">
     <property name="text">
      <string>Straft Left:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="downLabel">
     <property name="text">
      <string>Down:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QPushButton" name="takePictureHotkeyButtton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QLabel" name="takePictureLabel">
     <property name="text">
      <string>Take Picture:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="rollRightLabel">
     <property name="text">
      <string>Roll Right:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="rollRightHotkeyButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="backwardsLabel">
     <property name="text">
      <string>Backwards:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="backwardsHotkeyButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="speedIncreaseLabel">
     <property name="text">
      <string>Speed Increase:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="turnRightLabel">
     <property name="text">
      <string>Turn Right:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QPushButton" name="speedIncreaseHotkeyButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="turnRightHotkeyButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="6" column="8">
    <widget class="QPushButton" name="exitButton">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="forwardLabel">
     <property name="text">
      <string>Forward:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="forwardHotkeyButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="straftRightLabel">
     <property name="text">
      <string>Straft Right:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="pitchUpLabel">
     <property name="text">
      <string>Pitch Up:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pitchUpHotkeyButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QPushButton" name="turnLeftHotkeyButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="rollLeftHotkeyButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLabel" name="turnLeftLabel">
     <property name="text">
      <string>Turn Left:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="straftRightHotkeyButton">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="rollLeftLabel">
     <property name="text">
      <string>Roll Left:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="7">
    <widget class="QPushButton" name="defaultKeysButton">
     <property name="text">
      <string>Default Keys</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>pressed()</signal>
   <receiver>hotkeysWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
